app/controllers/assignments_controller.rb

```diff
@@ -894,12 +892,13 @@ def starter_file_group_file_data(starter_file_group)
   end
 
   def graders_options_params
-    params.require(:attribute)
-          .permit(assignment_properties_attributes: [
-            :assign_graders_to_criteria,
-            :anonymize_groups,
-            :hide_unassigned_criteria
-          ])
+    params.expect(attribute: [
```

Comment:
I've updated the Rubocop configuration for pre-commit, though to be honest I'm not sure why you ran into this issue. In any case, please do a pull from upstream master and then revert these changes. They should no longer be flagged by Rubocop.